pkg/executables/cmk.go

```diff
@@ -0,0 +1,265 @@
+package executables
+
+import (
+	"bytes"
+	"context"
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"path/filepath"
+	"time"
+
+	"github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+	"github.com/aws/eks-anywhere/pkg/filewriter"
+	"github.com/aws/eks-anywhere/pkg/logger"
+	"github.com/aws/eks-anywhere/pkg/retrier"
+	"github.com/aws/eks-anywhere/pkg/templater"
+)
+
+//go:embed config/cmk.ini
+var cmkConfigTemplate string
+
+const (
+	cmkConfigFileName = "cmk_tmp.ini"
+	maxRetriesCmk     = 5
+	backOffPeriodCmk  = 5 * time.Second
+)
+
+type Cmk struct {
+	writer     filewriter.FileWriter
+	executable Executable
+	retrier    *retrier.Retrier
```

Comment:
Are we using this? I can't find any references :)

Reply:
Not yet - will use this in the provider factory when integrating with the provider

Reply:
ok, do you might leaving a comment there indicating just that? just so we don't lose track of the reason why this is here :)